nf_core/modules/bump_versions.py

```diff
@@ -130,6 +145,8 @@ def bump_versions(
 
         self._print_results()
 
+        return nfcore_modules
```

Comment:
this allows me to test _which_ modules were updated in the new and existing tests